overrides/scripts/Earlygame.zs

```diff
@@ -210,19 +210,6 @@ recipes.addShaped(<gregtech:meta_item_1:32640>, [
 	[<ore:cableGtSingleTin>, <gregtech:meta_item_1:14197>,<gregtech:meta_item_1:14197>],
 	[<ore:cableGtSingleTin>, <gregtech:meta_item_1:32600>, <gregtech:meta_item_2:26197>]]);
 
-<simplefluidtanks:wrench>.displayName = "Multiblock Fluid Tank Wrench";
-<simplefluidtanks:tankitem>.displayName = "Multiblock Fluid Tank Block";
-<simplefluidtanks:valveitem>.displayName = "Multiblock Fluid Tank Valve";

```

Comment:
I'm not sure we want to remove the name changes. Perhaps it would be better to move the display name and tooltip overrides to the lang files like we have with other things.